### PR TITLE
removing yarn

### PR DESCRIPTION
**Pull request by Lorquas**
This removes yarn in favor of npm since some of you have desired to do this.  This PR represents a straight conversion.  with `npm config set repository` we don't have to maintain multiple lock files now.

### CODE REVIEW

File: cockpit/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "description": "a plugin for cockpit to manage subscriptions",
   "main": "index.js",
-  "repository": "git@github.com:rarebreed/subscription-manager.git",
```
Comment (Lorquas) [inline cockpit/package.json:6]: We should consider filling out this section fully, like where to open bugs and all the other project metadata...

Also everything in this file is listed as a devDependency , (previously added with `yarn add package --dev` from the README), this should really be reserved for test deps and transpilers and things not needed to actually use the project.

This overhaul would be a good item to make a card about.

File: .tito/cockpit-build.sh [hunk 2]
```diff
@@ -8,13 +8,8 @@ if [ -n "${USE_SUSE}" ] ; then
...
 
 echo "Generating cockpit dist-gzip..."
-if [ ! $(which yarn) ] ; then
-  echo "Unable to build cockpit, yarn not installed (see README)"
-  exit 1
-fi
 pushd cockpit
-yarn cache clean
-yarn install --frozen-lockfile
+npm ci
```
Comment (Lorquas) [inline .tito/cockpit-build.sh:12]: `npm ci` info > https://docs.npmjs.com/cli/ci.html
for more reproducible builds from the lockfile
we could also change this to `npm ci --cache $(mktemp -d)`  if we want to force re-download all the deps

File: cockpit/package.json [hunk 3]
```diff
@@ -67,5 +67,8 @@
     "hoek": "^4.2.1",
     "moment": "^2.19.3",
     "lodash": "^4.17.13"
+  },
+  "engines": {
+    "npm": "*"
```
Comment (Lorquas) [inline cockpit/package.json:72]: probably should lock this down to 5.10.0 or newer (i used 6.14.8)

File: .tito/cockpit-build.sh [hunk 4]
```diff
@@ -8,13 +8,8 @@ if [ -n "${USE_SUSE}" ] ; then
 fi
 
 echo "Generating cockpit dist-gzip..."
-if [ ! $(which yarn) ] ; then
-  echo "Unable to build cockpit, yarn not installed (see README)"
```
Comment (Lorquas) [inline .tito/cockpit-build.sh:12]: I don't know if you want this same nice message for npm itself.  You will get a 'npm command not found' error anyway...

**Comment by Lorquas** [comment]
The broken test is due to a flake8 error in master... would also like assistance in testing an actual cockpit from this build @jstavel

**Comment by jstavel** [comment]
I will test it @Lorquas

File: .tito/cockpit-build.sh (hunk 4 above)
Comment (cnsnyder) [reply .tito/cockpit-build.sh:12]: Yea that is fine.

File: cockpit/package.json (hunk 3 above)
Comment (cnsnyder) [reply cockpit/package.json:72]: I think that is a good idea, something we could even change in each of the support branches.

**Review by cnsnyder — APPROVED** [review]
Looks good to me! Thank you very much!

**Pull request merged**